hadoop-tools/hadoop-aws/src/site/markdown/tools/hadoop-aws/committers.md

```diff
@@ -180,20 +180,20 @@ and restarting the job.
 whose output is in the job attempt directory, *and only rerunning all uncommitted tasks*.
 
 
-This algorithm does not works safely or swiftly with AWS S3 storage because 
-tenames go from being fast, atomic operations to slow operations which can fail partway through.
+This algorithm does not work safely or swiftly with AWS S3 storage because
+renames go from being fast, atomic operations to slow operations which can fail partway through.
 
 This then is the problem which the S3A committers address:
 
-*How to safely and reliably commit work to Amazon S3 or compatible object store*
+>*How to safely and reliably commit work to Amazon S3 or compatible object store.*
```

Comment:
What are we quoting here?

Reply:
Fair point, it is not quoting anyone (or maybe Steve). I will let it join the previous sentence as I think that makes more sense.